command/operator_raft_join.go

```diff
@@ -139,10 +167,25 @@ func (c *OperatorRaftJoinCommand) Run(args []string) int {
 		return OutputData(c.UI, resp)
 	}
 
-	out := []string{}
-	out = append(out, "Key | Value")
-	out = append(out, fmt.Sprintf("Joined | %t", resp.Joined))
+	out := []string{
+		"Key | Value",
+		fmt.Sprintf("Joined | %t", resp.Joined),
+	}
 	c.UI.Output(tableOutput(out, nil))
 
 	return 0
 }
+
+func parseArg(raw string) (string, error) {
```

Comment:
Since they are technically flag values and not arg values, maybe rename it to `parseFlagVal` or something along those lines. We could also move this to `base_helper.go` (and maybe include a unit test for it).

Reply:
Good idea -- updated